plugins/services/src/js/structs/__tests__/Pod-test.js

```diff
@@ -211,6 +212,58 @@ describe("Pod", function() {
     });
   });
 
+  describe("#getTaskCount", function() {
+    it("should return 0 if there are no instances", function() {
+      const pod = new Pod({
+        spec: {
+          scaling: {
+            kind: "fixed",
+            instances: 1
+          }
+        },
+        instances: []
+      });
+
+      expect(pod.getTaskCount()).toEqual(0);
+    });
+
+    it("should return 1 if there is one running instance", function() {
+      const pod = new Pod({
+        spec: {
+          scaling: {
+            kind: "fixed",
+            instances: 1
+          }
+        },
+        instances: [{ status: PodInstanceState.STABLE }]
+      });
+
+      expect(pod.getTaskCount()).toEqual(1);
+    });
+
+    it("should return 1 if there is one running instance and other not-running ones", function() {
+      const possibleStates = Object.values(PodInstanceState).map(function(
+        state
+      ) {
+        return {
+          status: state
+        };
+      });
+
+      const pod = new Pod({
+        spec: {
+          scaling: {
+            kind: "fixed",
+            instances: 1
+          }
+        },
+        instances: possibleStates
+      });
+
+      expect(pod.getTaskCount()).toEqual(2);
```

Comment:
🤔 The test description says "should return 1...", but we assert 2?

Reply:
Great catch, will change that 👍

Reply:
I think Alfred still doesn't like this one.